Package.swift

```diff
@@ -99,6 +99,9 @@ let package = Package(
             /** Runs package tests */
             name: "swift-test",
             dependencies: ["Commands"]),
+        Target(
+            /** Shim tool to find test names on OS X */
+            name: "swiftpm-xctest-helper"),
```

Comment:
Is the bootstrap script happy with this? We don't need this tool to be built in stage1, but we do need to make sure the bootstrap script doesn't reject it.